pandas_gbq/tests/test_gbq.py

```diff
@@ -13,10 +13,13 @@
 from pandas import compat
 
 from pandas.compat import u, range
-from pandas import NaT, DataFrame
+from pandas import NaT, DataFrame, to_datetime
 from pandas_gbq import gbq
 import pandas.util.testing as tm
 from pandas.compat.numpy import np_datetime64_compat
+from google.cloud import bigquery
```

Comment:
you prob need a pytest.importorskip here

Reply:
Sorry, I'm not that familiar with pytest. Can you provide more guidance or perhaps someone else can let know what I should do? Otherwise, I can poke around when I have time, but that might not be for a while :(

Reply:
This becomes a hard dependency (right @jasonqng ?), so no skipping